### ISSUE TITLE
optimize dm02 for version 5

### ISSUE THREAD

**Issue by amygimma** [jarvisc1/comix_data_clean#43]
currently the spss file name is passed in to the save_spss_qs function, if we pass in the opened spss as a data table instead, we can loop through all of the country names within the same spss file, only opening and reading once.

**Comment by jarvisc1** [comment]
Agree. I think we have one issue where the first round came through as separate countries
